docs/getting-started.md

```diff
@@ -27,7 +27,7 @@ If you install it on the backend side, you may need to also install [`isomorphic
 
 ## Instantiation
 
-```js
+```ts
```

Comment:
Might be cool to have two sections one ts and one js. People usually look for one if they don't know typescript they won't know to remove the while ` as ERC725JSONSchema[]`. Or add a commented out line "use this for plain javascript". Also NOTE: import is only available inside of a "mjs" file when running it locally in node. So the "plain version" should probably also use "require" or specify that it has to be ".mjs".

Reply:
Or then no tabs but two code snippets examples